mantidproject/mantid
# Abins: Bose factors for improved simulation of temperature effects

**Description of work.**

Closes #34757 

The temperature contributions to incoherent inelastic structure calculation are overhauled to better align with methods described in the academic literature.

Currently Abins includes a <2n+1> occupation term [coth(x)] in the Debye-Waller (exponential) part, but this is applied at the measured frequency and squared.

Here a <n+1> (Bose) occupation is used for the mode displacement tensors (**B**) with <2n+1> occupations in the summation over mode displacements to form atomic displacement tensors (**A**). This is not energy-dependent after the sum has been made.

At this point the negative-energy-transfer transitions with occupation <n> are not included; this is under consideration but will require deeper reworking of binning steps which currently assume that only transitions within the visible results range are relevant. (This is violated if negative transitions are included in multi-phonon summations.)

This has several significant implications for Abins results and intermediate data:
- Overall results should be very similar at low temperature
- Overall results show significantly better agreement with other implementations for elevated temperatures
- The internal data structure collecting displacement tensors (PowderData) stores slightly different values (including these occupations) and also carries a copy of the mode <n+1> values so that it is possible to inspect or reverse the thermal effects of cached data. [This ability is not currently implemented; calculations at a new temperature will trigger a recalculation of the PowderData.]

**To test:**

Mantid Unit tests and systemtests have been updated to reflect the changed results and API tweaks.

Release notes are required and included; while objectively better, this is arguably a "breaking change" that affects established Abins results.

Below we verify that low-temperature effects are minimal and that agreement with other implementations is as good as expected.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
